app/controllers/forms/live_controller.rb

```diff
@@ -15,4 +15,16 @@ def show_pages
   def current_live_form
     Form.find_live(params[:form_id])
   end
+
+  def metrics_data
+    today = Time.zone.today
+    form_is_new = (today.to_date - Time.zone.parse(current_form.live_at).to_date).to_i < 1
+    # TODO: fetch real data from CloudWatch
```

Comment:
This should work fine, if we get an error from the cloudwatch service we might need to put some sort of handler here

Reply:
Have rebased onto your branch and made it read real data - included error handling as part of that.